lib/ansible/modules/cloud/vmware/vmware_guest.py

```diff
@@ -484,17 +483,19 @@ def set_powerstate(self, vm, state, force):
         facts = self.gather_facts(vm)
         expected_state = state.replace('_', '').lower()
         current_state = facts['hw_power_status'].lower()
-        result = {}
+        result = dict(
+            changed = False,
```

Comment:
I don't think it improves readability, especially not for multi-line dicts. But you're the boss.